base/floatfuncs.jl

```diff
@@ -123,19 +123,35 @@ round(::Type{T}, x::AbstractFloat, r::RoundingMode) where {T<:Integer} = trunc(T
 
 # NOTE: this relies on the current keyword dispatch behaviour (#9498).
 function round(x::Real, r::RoundingMode=RoundNearest;
-    digits::Union{Nothing,Integer}=nothing, sigdigits::Union{Nothing,Integer}=nothing, base=10)
+               digits::Union{Nothing,Integer}=nothing, sigdigits::Union{Nothing,Integer}=nothing, base::Union{Nothing,Integer}=nothing)
     isfinite(x) || return x
-    _round(x,r,digits,sigdigits,base)
+    if digits === nothing
+        if sigdigits === nothing
+            if base === nothing
+                # avoid recursive calls
+                throw(MethodError(round, (x,r)))
+            else
+                round(x,r)
+                # or throw(ArgumentError("`round` cannot use `base` argument without `digits` or `sigdigits` arguments."))
```

Comment:
decision needed here.

Reply:
Although specifying `base` without `digits` or `sigdigits` does not make much sense, I don't see any reason for disallowing it. The desired result is pretty unambiguous, after all.